man/bootctl.xml

```diff
@@ -377,6 +377,12 @@
         <listitem><para>Install binaries for all supported EFI architectures (this implies <option>--no-variables</option>).</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--efi-label=</option></term>
+        <listitem><para>Name of the entry added to the firmware's boot loader list. Defaults to <literal>Linux
```

Comment:
similar, here, call this "description", not "name" and "boot option list" instead of "boot loader list"